confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioSearchServletFilter.java

```diff
@@ -71,7 +80,8 @@ public void doFilter(
     final HttpServletRequest httpreq = (HttpServletRequest) servletRequest;
     // Saving a page or blogpost fires: PUT http://confluence-server:8090/rest/api/content/65603?status=draft
     if (httpreq.getRequestURI().startsWith("/rest/api/content/") &&
-        ("PUT".equals(httpreq.getMethod()) || "POST".equals(httpreq.getMethod()) || "DELETE".equals(httpreq.getMethod()))) {
+        ("PUT".equals(httpreq.getMethod()) || "POST".equals(httpreq.getMethod()) || "DELETE".equals(
```

Comment:
do you want to create a separate else-if case for DELETE with a different comment ?

Reply:
done